site/docs/hosting/cloudflare-workers-nodejs.md

```diff
@@ -316,6 +297,34 @@ Follow the instruction and input your bot token, your bot token will be uploaded
 You can change to whatever name you want for the environment variables, but keep in mind that you do the same in following steps.
 :::
 
+After we set `BOT_INFO` in `wrangler.toml` and added `BOT_TOKEN` as a secret, `index.ts` will throw an error because it lacks the definition of the `Env` type. To solve this problem, we need to do the following:
```

Comment:
Let's stick to our convention of having one sentence per line.
```suggestion
After we set `BOT_INFO` in `wrangler.toml` and added `BOT_TOKEN` as a secret, `index.ts` will throw an error because it lacks the definition of the `Env` type.
To solve this problem, we need to do the following:
```